src/Form/RadioGroup.js

```diff
@@ -60,7 +63,7 @@ class RadioGroup extends Component {
   };
 
   render() {
-    const { choices, error, horizontal, label, name, colorOn, colorOff, size } = this.props;
+    const { choices, error, horizontal, label, name, colorOn, colorOff, fontSize, iconSize, styles } = this.props;
```

Comment:
we could spread these final five props into something like `...checkboxProps` and spread onto `<Checkbox />` below

Reply:
Sounds good, will update.

Reply:
Thanks, updated.